trello_api.py

```diff
@@ -0,0 +1,92 @@
+from flask import session
+from classes import todoStatus, todoItem
+import os, requests, json, logging
+
+API_PREFIX = 'https://api.trello.com/1/'
+
+API_KEY = os.getenv('API_KEY')
+API_TOKEN = os.getenv('API_TOKEN')
+# API_PARAMS was getting appended with additional parameters, so had to set variable in each API function
+API_PARAMS = {'key': API_KEY, 'token': API_TOKEN}
+board = os.getenv('BOARD_ID')
+headers = {"Accept": "application/json"}
+
+
+def trelloGet(trelloPath):
+    API_PARAMS = {'key': API_KEY, 'token': API_TOKEN}
+    return requests.get(API_PREFIX + trelloPath, params=API_PARAMS).json()
+
+
+def get_trello_lists():
+    trelloLists = []
+    for item in trelloGet(f'boards/{board}/lists'):
+        list_data = todoStatus(
+            item['id'],
+            item['name']
+        )
+        trelloLists.append(list_data)
+        if list_data.status == "Not Started":
+            session['newItemId'] = list_data.trelloId
```

Comment:
This works, but it's a bit odd to store these values on the session. Cookies are for user specific data but this is constant for the instance of the app, as it's tied to the board that's in use. 

It would make a bit more sense to either set some constants when the app starts up or you could even get the list ID each time you need it.